svc/FetchService.js

```diff
@@ -78,9 +78,17 @@ export class FetchService {
             }
         }
 
-        const ret = await fetch(url, opts);
-        if (!ret.ok) throw Exception.requestError(opts, ret);
-        return ret;
+        try {
+            const ret = await fetch(url, opts);
+            if (ret && !ret.ok) throw Exception.requestError(opts, ret);
+            return ret;
+        } catch (e) {
+            if (e.message == 'Failed to fetch') {
```

Comment:
e has the following properties:
            console.log(e.name)  = TypeError
            console.log(e.message)  = Failed to fetch
            console.log(e.status) undefined
            console.log(e.stack) TypeError: Failed  to fetch

It seemed save to test on 'message'.   But message's value may be browser specific.  this is the value on Chrome.

Reply:
This thread: https://github.com/github/fetch/issues/310  suggests the error msg may vary by fetch implementation.